TombEngine/Game/control/los.cpp

```diff
@@ -136,31 +136,23 @@ bool GetTargetOnLOS(GameVector* origin, GameVector* target, bool drawTarget, boo
 						if ((Objects[item->ObjectNumber].explodableMeshbits & ShatterItem.bit) &&
 							LaserSight)
 						{
-							//if (!Objects[item->objectNumber].intelligent)
-							//{
+							if (item->ObjectNumber == ID_GUARD_LASER)
+							{
+								float angle = phd_atan(LaraItem->Pose.Position.z - item->Pose.Position.z, LaraItem->Pose.Position.x - item->Pose.Position.x) - item->Pose.Orientation.y;
```

Comment:
Type should probably be `short`, not `float`.

Reply:
fixed